techniques/system/common/1.0/common.st

```diff
@@ -3,7 +3,8 @@ bundle common system_common {
   vars:
     "directiveId"           string => "&TRACKINGKEY&";
     "sharedfiles_folder"    string => "&SHARED_FILES_FOLDER&";
-    "community_port"        string => "&COMMUNITYPORT&";
+    "cf_serverd_port"       string => "&COMMUNITYPORT&";
```

Comment:
&COMMUNITYPORT& is also used in cf-serverd.st shouldn't it be changed there as well?

Reply:
this is just a name change, &COMMUNITYPORT& already contains the proper port